spanner_orm/api.py

```diff
@@ -71,6 +71,11 @@ def run_write(cls, *args, **kwargs):
     """Wraps write and read-write queries in a transaction."""
     return cls._connection().run_in_transaction(*args, **kwargs)
 
+  @staticmethod
+  def delete(transaction, table_name, keyset):
+    _logger.debug('Delete table=%s keys=%s', table_name, keyset.keys)
+    transaction.delete(table=table_Name, keyset=keyset)
```

Comment:
there is a typo here. how is this passing test? can you add a test that catches that